src/main/resources/application.yaml

```diff
@@ -113,6 +113,8 @@ spring:
             loggerLevel: ${FEIGN_LOG_LEVEL:none}
 
 darts:
+  event-obfuscation:
+    enabled: ${EVENT_OBFUSCATION_ENABLED:false}
```

Comment:
Can you confirm this point from the ticket?
> Check the env var "EVENT_OBFUSCATION_ENABLED" exists in flux for the darts-api service staging environment, set to true. Ensure it is not set or set to false for all other environments.

Reply:
This is already set :)
https://github.com/hmcts/sds-flux-config/blob/master/apps/darts-modernisation/darts-api/stg.yaml